content/modules/ROOT/pages/module-private-docs.adoc

```diff
@@ -144,7 +144,7 @@ Use Retrieval-Augmented Generation (RAG) when you need to access a dynamic knowl
 
 On the other hand, choose fine-tuning when your task is specialized and you need precise control over the model's behavior. Fine-tuning is better suited for homogeneous data, offline or low-latency applications, and situations where security or compliance requires all data to be embedded within the model. It's also preferable when you have a well-defined use case with specific task requirements.
 
-TIP: See the xref:module-ilab.adoc[AI Model Fine-Tuning module^] for more information on fine-tuning models.
+TIP: See the xref:module-ilab.adoc[AI Model Fine-Tuning module,window=_blank] for more information on fine-tuning models.
 
```

Comment:
shouldn't it be the same thing?

Reply:
When using the `xref` macro, if you want to open in a new window you have to use `,window=_blank` as `^` doesn't work.

Reply:
makes sense!